packages/react-devtools-extensions/src/main.js

```diff
@@ -115,7 +115,20 @@ function createPanelIfReactLoaded() {
 
       const tabId = chrome.devtools.inspectedWindow.tabId;
 
-      registerDevToolsEventLogger('extension');
+      // TODO: after we upgrade to Manifest V3, chrome.tabs.query returns a Promise
+      // without the callback.
+      function getCurrentTab() {
+        return new Promise(resolve => {
+          chrome.tabs.query({active: true, currentWindow: true}, tabs => {
```

Comment:
Does using `chrome.tabs.url` or `chrome.tabs.pendingUrl` to get the URL not work?

Reply:
<img width="1356" alt="image" src="https://user-images.githubusercontent.com/2735514/176032219-1173331d-1574-4cbb-81a2-386af4b4ff5d.png">

They should be under the Tab Properties section. LMK if you can't find it we can VC and I can show you. 😊